src/libutil/meson.build

```diff
@@ -66,13 +66,21 @@ endif
 
 blake3 = dependency(
   'libblake3',
-  version: '>= 1.5.5',
+  version: '>= 1.7.0',
+  method : 'pkg-config',
 )
 deps_private += blake3
+tbb = dependency(
+  'tbb',
+  version: '>= 2021_11',
+  method : 'pkg-config',
+)
+deps_private += tbb
+add_project_arguments(['-DBLAKE3_USE_TBB=1'], language: 'cpp')
```

Comment:
Is this needed? Shouldn't this dep just be propagated from libblake?

Reply:
https://github.com/BLAKE3-team/BLAKE3/blob/master/c/CMakeLists.txt looking at this, I think some of the logic needs to be updated to also change pkg-config.

Basically I think the rationale in https://github.com/BLAKE3-team/BLAKE3/blob/master/c/CMakeLists.txt#L234C1-L237C65 is exactly right --- yes, you can use a tbb-supporint blake3 build without tbb, but it just runs the risk of linking multiple tbb builds, so better propagate the dep downstream unconditionally.

Reply:
Glad the upstream PR is merged, good luck with the rest!

Reply:
Btw. we switched in the meantime to unstable.

Reply:
I think we can rebase this?

Reply:
We can rebase but I was just waiting for the BLAKE3 changes to settle upstream before making further changes here.